src/app/core/config/models/config-submission-definition.model.ts

```diff
@@ -0,0 +1,19 @@
+import { ConfigObject } from './config.model';
+import { SubmissionSectionModel } from './config-submission-section.model';
+import { PaginatedList } from '../../data/paginated-list';
+import { ResourceType } from '../../shared/resource-type';
+
+export class SubmissionDefinitionModel extends ConfigObject {
```

Comment:
This class needs TypeDocs.  I realize it's a refactor of an old class, but this is a good opportunity to add in the missing TypeDocs at the class level.